src/redux/configureStore.js

```diff
@@ -1,5 +1,10 @@
 // Bookstore
-import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { combineReducers, legacy_createStore as createStore } from '@reduxjs/toolkit';
+
```

Comment:
- [ ] `createStore` and `combineReducers` should be imported from `'redux'` and not `'@reduxjs/toolkit'`. You can import configureStore from `'@reduxjs/toolkit'` but `createStore` should be imported from `'redux'`. Kindly implement this change. 👍

Reply:
Done